pkg/agent/controller/traceflow/packetin.go

```diff
@@ -176,15 +176,15 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*opsv1alpha1.Tracefl
 		obs = append(obs, *ob)
 	}
 
-	// Get drop table.
+	// Get table which is not allow. e.g. reject, drop.
```

Comment:
I personally think it's okay to keep as is here since the packet is going to be dropped eventually

Reply:
I was thinking to avoid confusion. But yea I can change it back.